tests/app/views/contexts/summary/test_answer.py

```diff
@@ -1,29 +1,39 @@
-from unittest import TestCase
-
 from app.views.contexts.summary.answer import Answer
+from tests.app.app_context_test_case import AppContextTestCase
 
 
-class TestAnswer(TestCase):
+class TestAnswer(AppContextTestCase):
     def test_create_answer(self):
         # Given
         answer_schema = {"id": "answer-id", "label": "Answer Label", "type": "date"}
         user_answer = "An answer"
 
         # When
-        answer = Answer(answer_schema, user_answer)
+        answer = Answer(
+            answer_schema,
+            user_answer,
+            "house-type",
+            None,
+            None,
+            "section-summary",
+        )
 
         # Then
         self.assertEqual(answer.id, "answer-id")
         self.assertEqual(answer.label, "Answer Label")
         self.assertEqual(answer.value, user_answer)
+        self.assertEqual(
+            answer.link,
+            "http://test.localdomain/questionnaire/house-type/?return_to=section-summary&return_to_answer_id=answer-id#answer-id",
```

Comment:
I think it's worth having this test separate and ensuring it works fine with a list name as well. I don't think we had a test for that.

Reply:
I'll include all elements and test the order of them all